overlord/snapstate/backend/link.go

```diff
@@ -122,7 +122,11 @@ func (b Backend) LinkSnap(info *snap.Info, model *asserts.Model, tm timings.Meas
 		switch info.InstanceName() {
 		case model.Kernel(), bootBase:
```

Comment:
same expectation

Reply:
for this one I've had to refactor things again so `Lookup` takes the `model`. No biggie.